.github/workflows/gitleaks.yml

```diff
@@ -0,0 +1,19 @@
+name: Gitleaks
+on:
+  pull_request:
+  push:
+  workflow_dispatch:
+  schedule:
+    - cron: "0 4 * * *" # run once a day at 4 AM
```

Comment:
Once a day for this repo is probably way overkill. I would say like once a week is good enough